packages/editor-ui/src/modules/userHelpers.ts

```diff
@@ -65,8 +65,8 @@ export const isAuthorized = (permissions: IPermissions, {currentUser, isUMEnable
 	}
 
 	if (permissions.allow) {
-		if (permissions.allow.um === isUMEnabled) {
-			return true;
+		if(permissions.allow.custom) {
+			return permissions.allow.custom();
```

Comment:
allow block is an OR rule... so it should allow user if any of the rules pass
```suggestion
		if(permissions.allow.custom && permissions.allow.custom()) {
			return true;
```

Reply:
Ok, updated.